cloudbuild-dev.yaml

```diff
@@ -17,33 +17,16 @@ steps:
       - |
         make audit
 
-  - name: python:3.11
-    id: "Check linting and code formatting"
-    entrypoint: sh
-    args:
-      - "-c"
-      - |
-        make lint
-
-  - name: python:3.11
-    id: "Run unit tests"
-    entrypoint: sh
-    args:
-      - "-c"
-      - |
-        make setup
-        make unit-test
-
   - name: docker
     id: "docker build and push"
     entrypoint: sh
     args:
       - "-c"
       - |
-        docker build -t "europe-west2-docker.pkg.dev/${PROJECT_ID}/sds/sds:${SHORT_SHA}" -t "europe-west2-docker.pkg.dev/${PROJECT_ID}/sds/sds:latest" .
+        docker build --cache-from "europe-west2-docker.pkg.dev/${PROJECT_ID}/sds/sds:latest" -t "europe-west2-docker.pkg.dev/${PROJECT_ID}/sds/sds:${SHORT_SHA}" -t "europe-west2-docker.pkg.dev/${PROJECT_ID}/sds/sds:latest" .
```

Comment:
Will it cause any issue if the SDS with :latest tag does not exist?

Reply:
It will continue with no cache if image isn't found, I tested this behaviour on my sandbox by deleting :latest tag from most recent image in artifact registry